packages/bbui/src/Button/Button.svelte

```diff
@@ -68,6 +68,10 @@
 </div>
 
 <style>
+  button {
+    width: 100%;
+  }
```

Comment:
Not sure about this - this could cause a few issues with client apps or elsewhere in the builder, unless this used to be here and was removed?